src/kendo.data.js

```diff
@@ -1149,6 +1149,12 @@ var __meta__ = { // jshint ignore:line
             },
             isnotnull: function(a) {
                 return "(" + a + " != null)";
+            },
+            isnullorempty: function(a) {
+                return "!" + a;

```

Comment:
Is it possible that a is a number?
if a is a number, this would be wrong.
when a is zero, this function return !0. that would be false
but the expect is true